bokehjs/src/coffee/util/dom_util.coffee

```diff
@@ -8,5 +8,11 @@ waitForElement = (el, fn) ->
       fn()
   interval = setInterval(handler, 50)
 
+# Add css_classes to an element
+add_css_classes = (el, css_classes) ->
+  for css_class in css_classes
+    el.addClass(css_class)
```

Comment:
You can add multiple classes with `addClass()`, no need for extra layer of indirection.